spec/student_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+describe Student do
+  before :each do
+    @student = Student.new(age: 14, name: 'Ced', parent_permission: true)
+  end
+
+  describe '#new' do
+    it 'returns a new student object' do
+      @student.should be_an_instance_of Student
+    end
+  end
```

Comment:
- [ ] Since we're aiming to cover all the methods in our classes, kindly write a test for the `play_hooky` method in your `student` class 👍